README.md

```diff
@@ -0,0 +1,5 @@
+# Questions about `pull-request`
+ 
+- Is there any example (repository, organization, ...) that shows the use of pull-requests in a classroom?
+- Do you think it would be easier to explain people with no technical background the use of git workflow using an IDE (like Fork) more than using terminal commands?
```

Comment:
Do your students come in without a technical background? Even so I would recommend the mental model / example / do it yourself pattern we use in the 🎥 .